modules/share/src/godotShareData.h

```diff
@@ -0,0 +1,53 @@
+/**************************************************************************/
+/*  godotShareData.h                                                      */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef GODOTSHAREDATA_H
+#define GODOTSHAREDATA_H
+
+#include "core/object/ref_counted.h"
+#include <version_generated.gen.h>
+
+class GodotShareData : public RefCounted {
+	GDCLASS(GodotShareData, RefCounted);
+
+protected:
+	static void _bind_methods();
+
+	GodotShareData *instance;
+
+public:
+	void shareText(const String &title, const String &subject, const String &text);
+	void shareImage(const String &path, const String &title, const String &subject, const String &text);
```

Comment:
```suggestion
	void share_text(const String &p_title, const String &p_subject, const String &p_text);
	void share_image(const String &p_path, const String &p_title, const String &p_subject, const String &p_text);
```

Same for the methods, argument names should have `p_` prefix.